src/settings/Settings.js

```diff
@@ -242,7 +242,7 @@ export const SETTINGS = {
             "room-account": _td("Enable URL previews for this room (only affects you)"),
             "room": _td("Enable URL previews by default for participants in this room"),
         },
-        default: true,
+        default: false,
```

Comment:
I'm probably failing to understand here, but doesn't this disable them by default everywhere?

Reply:
arg this is me failing to back out https://github.com/matrix-org/matrix-react-sdk/pull/2001/commits/01659af2498ef253d3853a2868b3547a62e37d52